packages/metastream-app/src/locale/pt-BR.ts

```diff
@@ -1,6 +1,7 @@
 export default {
   add: 'Adicionar',
   addMedia: 'Adicionar Midia',
+  addMediaCTA: 'Inice adicionando mídia',
```

Comment:
Typo
```suggestion
  addMediaCTA: 'Inicie adicionando uma mídia',
```

Not sure if "mídia" is the best portuguese word for this context.
PT: Na real não tenho certeza se "mídia" é a melhor palavra em português para esse context.